src/main/java/org/jabref/gui/JabRefDialogService.java

```diff
@@ -69,11 +70,11 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(JabRefDialogService.class);
     private static PreferencesService preferences;
 
-    private final Window mainWindow;
+    private static Window mainWindow;
```

Comment:
Why did you made this a static variable?

Reply:
This was done because `createDialog()` and `createDialogWithOptOut()` were both static, I've updated both methods to be non-static.